packages/fxa-auth-server/lib/senders/emails/templates/cadReminder/index.ts

```diff
@@ -2,39 +2,57 @@
-            <mj-text css-class="header-text">Here's your reminder to sync devices.</mj-text>
-            <mj-image css-class="sync-logo" alt="Devices" src="https://accounts-static.cdn.mozilla.net/other/graphic-laptop-mobile.png"></mj-image>
-          </mj-column>
-        </mj-section>
-        <mj-section>
-          <mj-column>
-            <mj-text css-class="primary-text">It takes two to sync. Syncing another device with Firefox privately keeps your bookmarks, passwords and other Firefox data the same everywhere you use Firefox.</mj-text>
-          </mj-column>
-        </mj-section>
-        ${button}
-        ${appBadges}
-        <mj-section>
-          <mj-column>
-            <mj-text css-class="secondary-text mt-6 mb-3">This is an automated email; if you received it in error, no action is required.For more information, please visit <span>Mozilla Support<span></mj-text>
-            <mj-text css-class="secondary-text">Mozilla. 2 Harrison St, #175, San Francisco, CA 94105</mj-text>
-            <mj-text css-class="secondary-text">Mozilla Privacy Policy</mj-text>
-            <mj-text css-class="secondary-text">Firefox Cloud Terms of Service</mj-text>
-          </mj-column>
-        </mj-section>
-      </mj-body>
-    </mjml>
-  `;
+  <mj-include path="./lib/senders/emails/css/cadReminder/index.css" type="css" css-inline="inline" />
+  <mj-section>
+    <mj-column>
+      <mj-text css-class="header-text sync-reminder">Here's your reminder to sync devices.</mj-text>
+      <mj-image css-class="sync-logo sync-description" alt="Devices" src="https://accounts-static.cdn.mozilla.net/other/graphic-laptop-mobile.png"></mj-image>
+    </mj-column>
+  </mj-section>
+  <mj-section>
+    <mj-column>
+      <mj-text css-class="primary-text">It takes two to sync. Syncing another device with Firefox privately keeps your bookmarks, passwords and other Firefox data the same everywhere you use Firefox.</mj-text>
+    </mj-column>
+  </mj-section>
+  ${button}
+  ${appBadges}
+  ${automatedEmailNoAction}
+`;
 };
+
+// Mjml helpers to add data-l10n attributes to the html elements
+const addDataL10nId = (path: string, id: string) => {
+  return `
+  <mj-selector path="${path}">
+    <mj-html-attribute name="data-l10n-id">${id}</mj-html-attribute>
+  </mj-selector>
+`;
+};
+
+const addDataL10nName = (path: string, name: string) => {
+  return `
+  <mj-selector path="${path}">
+      <mj-html-attribute name="data-l10n-name">${name}</mj-html-attribute>
+    </mj-selector>
+`;
+};
+
+export const localizationAttributes = `
+  <mj-html-attributes>
+    ${addDataL10nId('.header-text div', 'sync-reminder')}
+    ${addDataL10nId('.primary-text div', 'sync-description')}
+    ${addDataL10nId('.automated-text div', 'automated-email-text')}
+    ${addDataL10nId('.install-device-link div', 'another-desktop-device')}
+    ${addDataL10nId('.install-desktop-device-link div', 'another-device')}
+    ${addDataL10nId('.privacy-url div', 'footer-privacy-url')}
```

Comment:
I know the attribute names are looking weird, but I couldn't think about better ones. It'd be nice if we can set a convention for naming them

Reply:
Hrm, I know you did this way since MJML doesn't allow custom attributes, but I think we had said before (maybe just once in our 3-way group chat) that we could potentially wrap text strings in `<span>` to apply the custom data-attribute to... I don't like the idea of extra DOM elements so I wondered if there was a better way, but doing this per template/per line to be extracted seems a little confusing/cumbersome (especially given the weirdness around MJML adding those `div`s or table cells). I don't necessarily have an alternative outside of suggesting `span`, but if we do keep this, we probably want to file a follow up ticket if even to just investigate how else we can do this.